tools/stimulus/functions-R.yaml

```diff
@@ -572,7 +572,7 @@ igraph_hub_and_authority_scores:
     # in the argument list.
     DEPS: weights ON graph, hub ON graph V(graph), authority ON graph V(graph)
     PARAMS: GRAPH graph, OUT VERTEX_QTY hub, OUT VERTEX_QTY authority,
-        OUT REAL value, BOOLEAN scale=True, EDGEWEIGHTS weights=NULL,
+        OUT REAL value, BOOLEAN scale=True, OPTIONAL EDGEWEIGHTS weights,
```

Comment:
Let's keep:

```suggestion
        OUT REAL value, BOOLEAN scale=True, OPTIONAL EDGEWEIGHTS weights=NULL,
```

Reply:
I think you should not re-add `OPTIONAL`, only `NULL`. At this moment, it won't make a difference.

Reply:
Fixed in last commit.